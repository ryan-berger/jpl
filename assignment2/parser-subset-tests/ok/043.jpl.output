(StmtCmd (ReturnStmt (FloatExpr 162548)))
Compilation succeeded
