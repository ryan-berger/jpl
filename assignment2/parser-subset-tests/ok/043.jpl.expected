(StmtCmd (ReturnStmt (FloatExpr 162548)))
